index.bs

```diff
@@ -111,12 +111,26 @@ she downloads the photos and posts them to the NAS device.
 
 ## UC-07: Secure offline communication for home automation ## {#uc-07}
 
-A user sets up a home automation gateway is under normal circumstances using
-HTTPS to securely accept commands via a remote server. In some cases the
-gateways internet connection is interrupted but local communication between a
-wall mounted control surface, app, or similar should stil be given. This becomes
-very important when we consider that devices that can be controlled this way
-include door locks and security cameras.
+A user wants to set up a new home device (e.g., a Wi-Fi access point/router,
+home automation gateway) in his/her home using the device web interface.
+User typically connects the device with a laptop or a smartphone and then
+launches a given URL (e.g., https://home-router.local, or https://192.168.1.1)
+on the laptop or smart phone’s web browser. Since the new device does not have
```

Comment:
My suggestion is: "User typically opens a given URL for the device (e.g., https://home-router.local, or https://192.168.1.1) in a laptop or smart phone’s web browser."